runner.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python
+
+import sys
+assert sys.version_info >= (3, 9), "Use Python 3.9 or newer."
+
+import os
+import socket
+import optparse
+import datetime
+import shutil
+import subprocess
+import time
+import signal
+import json
+from typing import Union, Tuple
+
+'''
+The default flags depending on the hostname.
+'''
+DEFAULT_SETTINGS = {
+    'futharkhpa01fl.unicph.domain': {
+        'name': 'A100',
+        'labels': 'cuda,opencl,multicore',
+    },
+    'futharkhpa02fl.unicph.domain': {
+        'name': 'MI100',
+        'labels': 'opencl,multicore',
+    },
+}
+
+'''
+These are the commands used for installing the self-hosted runner. You may wish to update these
+commands with commands from the download box on the website.
+https://github.com/{organization}/{repository}/settings/actions/runners/new
+'''
+INSTALLATION = \
+'''curl -o actions-runner-linux-x64-2.299.1.tar.gz -L https://github.com/actions/runner/releases/download/v2.299.1/actions-runner-linux-x64-2.299.1.tar.gz
+echo "147c14700c6cb997421b9a239c012197f11ea9854cd901ee88ead6fe73a72c74  actions-runner-linux-x64-2.299.1.tar.gz" | shasum -a 256 -c
+tar xzf ./actions-runner-linux-x64-2.299.1.tar.gz
+rm actions-runner-linux-x64-2.299.1.tar.gz'''
+
+'''
+The folder where the runner files are stored.
+'''
+RUNNER_FOLDER = 'actions-runner'
```

Comment:
This should be the current directory (i.e. `.`).

Reply:
I have a question about this, I chose to do this since the script removes the `RUNNER_FOLDER` and its contents when the `--remove` flag is used (I now realize I forgot to mention this in `self-hosted-runner.md`). Would this not make it problematic having `RUNNER_FOLDER='.'`?

Reply:
It would certainly make it more dangerous to run.  Let's keep it the way it is.